app/coffeescript/map.coffee

```diff
@@ -27,10 +28,6 @@ define [
           draggable: true
           panControl: false
       markers:
-        mapPin:        "/assets/nonsprite/map/map_pin_red4.png"
```

Comment:
thanks for the tip. I will do that in a future Pr that has pins working in the demo.